src/app/components/named-entity/named-entity.component.ts

```diff
@@ -20,6 +23,11 @@ export class NamedEntityComponent implements OnInit {
   toggleContent() {
     if (this.inList) {
       this.contentOpened = !this.contentOpened;
+      if (this.contentOpened) {
+        if (this.entityDetails.activeId === 'info' && this.entity.content.length === 0) {
+          this.entityDetails.select('occurrences');
+        }
+      }
```

Comment:
This shouldn't be inside `toggleContent`.
Try to make this change as a callback of the event `contentOpened is true`